plugins/download.py

```diff
@@ -122,6 +129,34 @@ def _download_source(self, pkg_specs):
         locations = sorted([pkg.localPkg() for pkg in pkgs])
         return locations
 
+    def _download_debuginfo(self, pkg_specs):
+        """Download debuginfo packages to dnf cache."""
+        dbg_pkgs = set()
+        q = self.base.sack.query()
+        q = q.available()
+
+        for pkg in self._get_packages(pkg_specs):
+            for dbg_name in [dnfpluginscore.lib.package_debug_name(pkg),
+                            dnfpluginscore.lib.package_source_debug_name(pkg)]:
+                dbg_available = q.filter(
+                                        name=dbg_name,
+                                        epoch=int(pkg.epoch),
+                                        version=str(pkg.version),
```

Comment:
> any reason why you convert it to str? 

Yeah, copy & paste :). I've taken this directly from `debuginfo-install.py` at the time, see _dbg_available. I will submit an updated PR in a second, but have in mind it isn't tested b/c of the test suite changes in PR #113. Please merge that one first.